src/app/tile/tile.component.html

```diff
@@ -1,4 +1,5 @@
 <div class="tile" [class.face-down]="isFaceDown" [class.face-up]="isFaceUp" (click)="switchFace()">
+  <aar-tile-back *ngIf="isFaceDown" [imageName]="imageName"></aar-tile-back>  
```

Comment:
Maybe a test to confirm that an aar-tile-back is shown when face down. Would make more sense to write before doing the chang tho.